packages/common/aws.js

```diff
@@ -261,13 +261,23 @@ exports.downloadS3File = (s3Obj, filepath) => {
 /**
 * Get an object header from S3
 *
-* @param {string} bucket - name of bucket
-* @param {string} key - key for object (filepath + filename)
+* @param {string} Bucket - name of bucket
+* @param {string} Key - key for object (filepath + filename)
 * @returns {Promise} - returns response from `S3.headObject` as a promise
 **/
+exports.headObject = (Bucket, Key) =>
```

Comment:
You should note the new functionality in the changelog.

Reply:
Which new functionality?

Reply:
Maybe the new `getObjectSize` helper?